src/common/padPlayer.js

```diff
@@ -0,0 +1,30 @@
+import { Players } from 'tone'
+import { padsData } from './padsData'
+import { createKeyFromString } from './utils'
+
+const padUrls = createPadUrlObjects(padsData)
+
+export const padPlayer = new Players(padUrls).toMaster()
```

Comment:
Maybe figure out the pad to play in this file and don't export `padPlayer` at all, so your App is not directly coupled to `tone`